lib/tree-view.coffee

```diff
@@ -276,6 +276,7 @@ class TreeView extends View
 
     @roots = for projectPath in atom.project.getPaths()
       stats = fs.lstatSyncNoException(projectPath)
+      continue unless stats
```

Comment:
This was easier to comment out to verify the specs were trustworthy. :p But okay.